corehq/apps/app_manager/models.py

```diff
@@ -5386,17 +5388,13 @@ def fetch_jar(self):
 
     def make_build(self, comment=None, user_id=None, previous_version=None):
         copy = super(ApplicationBase, self).make_build()
+        copy._all_files = self._all_files
         if not copy._id:
             # I expect this always to be the case
             # but check explicitly so as not to change the _id if it exists
             copy._id = uuid.uuid4().hex
 
-        force_new_forms = False
```

Comment:
@calellowitz @orangejenny @dannyroberts Maybe one of you has context with the above question?

Reply:
bump.  I'm picking this up from Nick and would also like to know the answer to this

Reply:
I have to say it's been a while, but both from this description and from looking at this code, that looks like a reasonable thing to do, and I don't think it could end in a different result.

Reply:
nifty, thanks!